test/spec/app_spec.rb

```diff
@@ -457,48 +457,49 @@
       expect(displayed_name.include? "EDIT").to be(true)
     end
 
-  # This tests is unreliable
-  # Todo: Methods that deal react with hidden menus
-  #
-  # ccx 2017-10-13
-=begin
-    it "should add and remove accounts to sources", bin6: true  do
+    it "should add and remove accounts to sources", bin6: true do
       api_create_team_project_and_source_and_redirect_to_source('GOT', 'https://twitter.com/GameOfThrones')
-      #sleep 5
-      element = wait_for_selector("source__edit-button", :class)
+      wait_for_selector("source__tab-button-account",:class)
+      element = wait_for_selector("source__edit-button",:class)
       element.click
-      #sleep 3
+      sleep 1
```

Comment:
do we still need the `sleep`s in these tests?

Reply:
Because between click in the link to add and start to insert the information the browser lost it self if it is immediate .

Reply:
I made a new test removing the sleeps and fails :(